api/src/middlewares/firebaseAuthenticate.js

```diff
@@ -0,0 +1,15 @@
+import { fetchByEmail } from '../services/userService';
+
+export const authenticateUser = async (req, res, next) => {
+  try {
+    const user = await fetchByEmail(req.params.email);
+
+    req.uid = user.uid;
+
+    next();
+  } catch (err) {
+    console.error(err);
```

Comment:
remove this

Reply:
done [database-integrate 81d7e81] database-integrate:#7 - remove console